com.vagueentertainment.Framed.yml

```diff
@@ -0,0 +1,75 @@
+app-id: com.vagueentertainment.Framed
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: framed
+modules: 
+  
+  - name: framed
+    buildsystem: simple
+    build-commands:
+      - install -D -m755 Framed.x86_64 /app/bin/Framed.x86_64
+      - install -D Framed.pck /app/bin/Framed.pck
+      - cp -r HoneyComb-redistributable /app/bin/HoneyComb-redistributable
+      - install -D -m755 framed.sh /app/bin/framed
+      - desktop-file-edit --remove-key=PrefersNonDefaultGPU $FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=$FLATPAK_ID $FLATPAK_ID.desktop
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 $FLATPAK_ID.appdata.xml /app/share/appdata/$FLATPAK_ID.appdata.xml
+            
+    sources:
+      - type: archive
+        url: https://github.com/VagueEntertainment/Framed/archive/refs/tags/v0.3.tar.gz
+        sha256: "1558912caf5d83347be02a598b59fa13e09ca758c1a35d18a4d407784292e667"
+        
+      - type: archive
+        url: https://github.com/VagueEntertainment/HoneyComb-redistributable/archive/refs/tags/0.1.1.tar.gz
+        sha256: "69177aaa166089ed14a41c1fe673abfcd08d0f57d2bdb62f524043d1e04ed5dc"
+        dest: HoneyComb-redistributable
+      
+      - type: script
+        dest-filename: framed.sh
+        commands:
+          - export APPDATA="$XDG_DATA_HOME"
+          - export XDG_DATA_HOME="$APPDATA"
```

Comment:
isn't there a circular reference making this line totally unnecessary?

Reply:
Yeah, this is rather silly. I was having issues with the flatpak writing to the users home directory outside confinement. I can probably get rid of the first line. Will test and resolve.

Reply:
With `--filesystem=home:ro` you probably can have --persist for those specific directories, until it's fixed upstream.